Assets/SEE/XR/Rotator.cs

```diff
@@ -176,12 +173,24 @@ private void GetRotationDistance(float currentAngle)
         }
 
         /// <summary>
-        /// Calculates the current rotation angle of the controller.
+        /// Checks if the rotation was significant enough (bigger than the angleTolerance).
```

Comment:
Name should be explicitly referenced:
```suggestion
        /// Checks if the rotation was significant enough (bigger than the <paramref name="angleTolerance"/>).
```